src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -239,7 +239,7 @@ public static class TransformerChain
     {
```

Comment:
Now that I think about it, I suspect that this entire class can be made internal. We should expose these methods in `MLContext` if we need them, and they are not there yet. Please let me know if I am wrong.

Reply:
That's all fine, but, maybe we could internalize things unrelated to the issue and this PR in subsequent PRs?

Reply:
So to be clear, I don't really intend to do this. Unless I literally can't get a signoff any other way.